src/interfaces.ts

```diff
@@ -67,6 +67,7 @@ export class MangoDepositoryAccount {
   netQuoteMinted!: BN; // i128
   quoteMintAndRedeemFee!: number; // u8
   totalQuoteMintAndRedeemFees!: BN; // u128
+  disabled!: boolean;
```

Comment:
could it say `mintingDisabled` rather than `disabled` for clarity ?

Reply:
yes, that's a better one, updated on both program/client